edx_sga/sga.py

```diff
@@ -228,14 +228,21 @@ def student_state(self):
         }
 
     def staff_grading_data(self):
-        def get_student_data():
+        def get_student_data(enrolled_students=[]):
```

Comment:
Just so you know, this is a dangerous game you're playing http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

Reply:
Ferenca approves